protocols/grpc/agent.proto

```diff
@@ -159,3 +162,26 @@ message Route {
 	string gateway = 2;
 	string device = 3;
 }
+
+message EventsRequest {
+	string container_id = 1;
+	uint32 PID = 2;
+	google.protobuf.Timestamp timestamp = 3;
+}
+
+message ContainerTermination {
+	string container_id = 1;
+	int32 exit_code = 2;
+}
+
+message ProcessTermination {
+	string process_id = 1;
+	int32 exit_code = 2;
+}
+
+message Event {
+	oneof event {
+		ContainerTermination container_termination = 1;
```

Comment:
I guess `ContainerTermination`  will be sent when all processes of container are terminated?

Is this equal to "receive ProcessTermination event of all processes"? It looks like it can be composed of multiple "ProcessTermination" event